### ISSUE TITLE
Add COS Fields GF (ON PROD)

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#7465]
### We're adding 8 folder infos to the GF folder.

**Stakeholder:** Robert Alv, Darren

**Acceptance Criteria**

- All changes/additions mentioned below are for the GF folder in AMANDA, with subtype 'ATD'.
- The infos should not display on the GF folder when work type is chosen as 'Dev Assessment'.

**Info fields to be added:**

- [ ] 'Fund', 
- [ ] 'Department', '
- [ ] Unit', 
- [ ] 'Object', 
- [ ] 'Work Order #' 
- [ ] 'Sub Project ID #' 
- [ ] COS Partner Department? (created in #7478)
- [ ] 'Partner Department Name' (created in #7478)

**Comment by TracyLinder** [comment]
@Nadin-Nader I'd like LIT to start building this out during this sprint.

**Comment by TracyLinder** [comment]
@Lpriya-LIT here is clarification:

These infos should only populate when:
- Subtype is ATD
- And Work type is Null, or work type is not Pre-Dev Assesment

Is this possible?

**Comment by Lpriya-LIT** [comment]
@TracyLinder Yes it is possible, will work on it

**Comment by Nadin-Nader** [comment]
@Lpriya-LIT This has been send to the DevOps CM Team to deploy to **_DEV_** Under **SCTASK0179115**

**Comment by Nadin-Nader** [comment]
**Status:**

Waiting on the DevOps CM Team to deploy to **_DEV_** Under **SCTASK0179115**

**Comment by TracyLinder** [comment]
When UAT is ready, our SME is Robert Alvarez.

**Comment by Nadin-Nader** [comment]
@Lpriya-LIT this has been deployed to DEV. Please UAT or send me a demo of it on DEV

**Comment by Lpriya-LIT** [comment]
Verified in Dev.
@Nadin-Nader As per the AC , the info questions will be displayed only for ATD subtype 
And Work type is Null, or work type is not Pre-Dev Assessment

**Comment by Lpriya-LIT** [comment]
![image.png](https://images.zenhubusercontent.com/611eac1954217f265acc0499/83a6f9f1-afe0-488a-855c-963749c87d90)

**Comment by Nadin-Nader** [comment]
@Lpriya-LIT I have submitted this to DevOps CM Team for **_TEST_** deployment under **SCTASK0179766**

**Comment by Nadin-Nader** [comment]
**Status:** 
Waiting on DevOps CM Team to deploy to **_TEST_** under **SCTASK0179766**

**Comment by Lpriya-LIT** [comment]
@Nadin-Nader Deployed to Test , verified . Changes look good

**Comment by Nadin-Nader** [comment]
@Lpriya-LIT Spoke to Tracy and she said lets get this to PROD if the fields are good to go on TEST.
 I am going to create a new ticket  to add the validation to the GF COS Fields but for after the code freeze.

**Comment by Nadin-Nader** [comment]
**Status:**

Ready for **PROD** Deployment

**Comment by Nadin-Nader** [comment]
@Lpriya-LIT 
**Status:**
This has been sent to the DevOps CM Team for deployment to **_PROD_** under **SCTASK0181711**

**Comment by Nadin-Nader** [comment]
@Lpriya-LIT @TracyLinder this has been deployed to **PROD**. I will send an email to the stake holders and hopefully get the approval to close Monday if no issues arise.

**Comment by Nadin-Nader** [comment]
Email to Stakeholders:

> Hi Darren and Robert,
> 
> Hope you are both doing well and having a wonderful holiday. I was emailing to inform you that the Cost Of Service (COS) fields are now on the GF folder on AMANDA PROD. I will go ahead and keep this open until Monday COB after which I will close if there are no issues with it.
> 
> If you have questions, please feel free to email me or send a Teams message.
> 
> Respectfully,
> 
> Nadin A. Nader (He/Him/His)
> Product Delivery Manager
> Data and Technology Services
>

**Comment by Nadin-Nader** [comment]
**Status:**

Will close Monday by COB if no issues arise

**Comment by Nadin-Nader** [comment]
Closing card, as no issues have been reported happening on PROD

**Comment by TracyLinder** [comment]
Action | Date | Duration
-- | -- | --
Opened | 10-15 | 0
Assigned to BA | 10-18 | 3
BA Done | 10-20 | 2
Dev Began | 11-3 | 14
Promoted to PROD | 11-25 | 22

**Total days from open to PROD: 41**
